frontend-cert/js-projects/library-manager/script.js

```diff
@@ -0,0 +1,97 @@
+// Initialize the library with some books – we can make campers fill in the first two, then supply the rest ourselves.
+const library = [
+  {
+    title: 'Battle Cry of Freedom: The Civil War Era',
+    author: 'James M. McPherson',
+    about: 'A book about the American civil war',
+    pages: 904,
+  },
+  {
+    title: 'Atomic Habits',
+    author: 'James Clear',
+    about:
+      'A practical book about discarding bad habits and building good ones',
+    pages: 320,
+  },
+  {
+    title: '1776',
+    author: 'David McCullough',
+    about:
+      'A book about those who marched with George Washington during the Declaration of Independence',
+    pages: 386,
+  },
+  {
+    title: 'The Embedded Entrepreneur',
+    author: 'Arvid Kahl',
+    about: 'A book focusing on how to build an audience-driven business',
+    pages: 308,
+  },
+  {
+    title: 'Congo: The Epic History of a People',
+    author: 'Adam Hochschild',
+    about:
+      'A book about the people of Congo and the colonization of their land by King Leopold',
+    pages: 639,
+  },
+  {
+    title: 'Precolonial Black Africa',
+    author: 'Cheikh Anta Diop',
+    about: 'A book about the analysis of precolonial African states',
+    pages: 240,
+  },
+  {
+    title: 'How Europe Underdeveloped Africa',
+    author: 'Walter Rodney',
+    about:
+      'A book detailing how European countries divided Africa between themselves and exploited it',
+    pages: 312,
+  },
```

Comment:
I vote we remove this book.

It looks like a good book but unfortunately people are sensitive nowadays.
So it is best to be on the safe side here

Reply:
I see. I will remove all the historical books there.